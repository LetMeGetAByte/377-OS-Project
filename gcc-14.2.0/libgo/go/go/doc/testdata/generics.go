version https://git-lfs.github.com/spec/v1
oid sha256:29959065a1382d67ed34e30caddcd314b0fe16ca1446665dc48a9c45ab0a8501
size 2117
