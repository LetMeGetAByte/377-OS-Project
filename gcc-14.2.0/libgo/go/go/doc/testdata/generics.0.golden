version https://git-lfs.github.com/spec/v1
oid sha256:1f13e93ec7def29294f6ba4928164b8cceb393efaa77bb77d874337564f5159b
size 1883
